cli/cmd/plugin/ui/web/tanzu-ui/src/components/ClusterSettings.tsx

```diff
@@ -41,15 +47,22 @@ function ClusterSettings(props: ChildProps | any) {
             <div cds-layout="horizontal gap:lg align:vertical-center">
                 <CdsInput>
                     <label>CLUSTER NAME</label>
-                    <input placeholder="CLUSTER NAME" 
+                    <input
+                        placeholder="CLUSTER NAME"
                         {...register('CLUSTER_NAME', { required: true })}
-                        defaultValue={state.data['CLUSTER_NAME']||''} onBlur={() => blurHandler()}/>
-                    { errors['CLUSTER_NAME'] && <CdsControlMessage status="error">{errors['CLUSTER_NAME'].message}</CdsControlMessage> }
+                        defaultValue={state.data['CLUSTER_NAME'] || ''}
+                        onBlur={() => blurHandler()}
+                    />
+                    {errors['CLUSTER_NAME'] && <CdsControlMessage status="error">{errors['CLUSTER_NAME'].message}</CdsControlMessage>}
```

Comment:
I see it took out the space between { and "errors". I prefer the space, but I get we may have limited choices